app/controllers/regulatory-attachments/edit.js

```diff
@@ -279,4 +287,43 @@ export default class EditController extends Controller {
     await documentContainer.save();
     this._editorDocument = editorDocument;
   });
+
+  get documentSnippetListIds() {
+    return (
+      this.editor
+        .getDocumentAttribute(SNIPPET_LISTS_IDS_DOCUMENT_ATTRIBUTE)
+        ?.split(',')
+        .filter(Boolean) ?? []
+    );
+  }
+
+  set documentSnippetListIds(snippetIds) {
+    this.editor.setDocumentAttribute(
+      SNIPPET_LISTS_IDS_DOCUMENT_ATTRIBUTE,
+      snippetIds.join(','),
+    );
+    this.assignedSnippetListsIds = snippetIds;
+  }
+
+  setDocumentContainerSnippetLists = task(async (snippetIds) => {
+    if (!snippetIds || !snippetIds.length) {
+      this.documentSnippetListIds = [];
+      this.model.documentContainer.snippetLists.setObjects([]);
+
+      return this.save.perform();
+    }
+
+    const snippetLists = await this.store.query('snippet-list', {
+      filter: {
+        ':id:': snippetIds.join(','),
+      },
+      include: 'snippets',
+      sort: 'created-on',
```

Comment:
this doesn't do anything
I missed it last time, but the modal that renders the list selection is a plugin component
it queries the lists on its own, and the sorting needs to happen there